.github/actions/combine-results/action.yaml

```diff
@@ -21,11 +21,11 @@ runs:
       echo "Summary:"
       failed="false"
       while IFS=" " read -r job result; do
-        if [[ "$result" != "success" ]]; then
+        if [[ "$result" != "failure" ]]; then
```

Comment:
what are the other possible statues?

Reply:
Excellent point. It looks like the other possible status is "canceled." I updated so that the it only considers success if the job is either a "success" or "skipped."